### DISCUSSION TITLE
How do I add crates? I am not able to import `use glob::glob`

### DISCUSSION THREAD

**Discussion by JanakyMurthy** [evcxr/evcxr#191] [Q&A] [answered]
I get the following error when I try to `use glob::glob`. How do I fix this?

```
use of undeclared crate or module `glob`
unresolved import `glob`
```

**Comment by bjorn3** [comment, marked as answer]
You need to first add it as dependency using something like `:dep glob="0.3.0"`.

**Reply by thomcc** [reply]
Note that you can also do `:dep glob` and it will pull in the latest version (or, will be equivalent to `:dep glob="*"`).
